examples/adwords/adwords_appengine_demo/README.md

```diff
@@ -86,7 +66,7 @@ prepared your OAuth2 Credentials via the Google Developers Console.
 
 1. You can now deploy this project by running the following command:
 
-    `python $APPENGINE_SDK_DIR/appcfg --oauth2 update .`
+    `gcloud app deploy app.yaml -v v1`
```

Comment:
Maybe suggesting a local deploy via `dev_appserver.py` would be better for purposes of getting users started. It might not be a bad idea, as there will be less for the user to set up in order to get this working. We can suggest that users configure their own cloud project when they're ready to deploy.

Reply:
Configuring the environment to execute it with 'dev_appserver.py' is the logical first step, it would be nice to reflect developers workflows in the documentation by approaching it in the same order.

However parts of the existing documentation are tangled with instructions that lead to a deployment and this would require more time to tests and review as more components are required to run `dev_appserver.py` if memory serves.

Reply:
Understood. I think you can leave it as-is then for now. I can modify this further to add steps for dev_appserver.